package.json

```diff
@@ -74,7 +75,7 @@
   },
   "devDependencies": {
     "@graphprotocol/client-cli": "3.0.0",
-    "@graphprotocol/graph-ts": "0.31.0",
+    "@graphprotocol/graph-ts": "^0.31.0",
```

Comment:
Hey, thanks for fixing the version, but you need to also remove the `^` else running `bun update` will cause an update of this package